### ISSUE TITLE
No dist folder in @nestjs/mapped-types. Unable to run the server.

### ISSUE THREAD

**Issue by slowestmonkey** [nestjs/nest#5412]
## Bug Report

Unable to run the server after mapped-types package installation and usage of PickType, PartialType decorators.

## Current behavior

```
[12:26:00 AM] Starting compilation in watch mode...

[12:26:13 AM] Found 0 errors. Watching for file changes.

internal/modules/cjs/loader.js:969
  throw err;
  ^

Error: Cannot find module './dist'
Require stack:
- <my_path>\node_modules\@nestjs\mapped-types\index.js
- <my_path>\node_modules\@nestjs\swagger\dist\type-helpers\intersection-type.helper.js
- <my_path>\node_modules\@nestjs\swagger\dist\type-helpers\index.js
- <my_path>\node_modules\@nestjs\swagger\dist\index.js
- <my_path>\node_modules\@nestjs\swagger\index.js
- <my_path>\dist\user\user.controller.js
- <my_path>\dist\user\user.module.js
- <my_path>\dist\auth\auth.module.js
- <my_path>\dist\app.module.js
- <my_path>\dist\main.js
    at Function.Module._resolveFilename (internal/modules/cjs/loader.js:966:15)
    at Function.Module._resolveFilename (<my_path>\node_modules\tsconfig-paths\lib\register.js:75:40)
    at Function.Module._load (internal/modules/cjs/loader.js:842:27)
    at Module.require (internal/modules/cjs/loader.js:1026:19)
    at require (internal/modules/cjs/helpers.js:72:18)
    at Object.<anonymous> (<my_path>\node_modules\@nestjs\mapped-types\index.js:6:10)
    at Module._compile (internal/modules/cjs/loader.js:1138:30)
    at Object.Module._extensions..js (internal/modules/cjs/loader.js:1158:10)
    at Module.load (internal/modules/cjs/loader.js:986:32)
    at Function.Module._load (internal/modules/cjs/loader.js:879:14) {
  code: 'MODULE_NOT_FOUND',
```

## Environment

```
"dependencies": {
        "@nestjs/common": "^7.4.4",
        "@nestjs/config": "^0.5.0",
        "@nestjs/core": "^7.4.4",
        "@nestjs/jwt": "^7.1.0",
        "@nestjs/mapped-types": "^0.1.0",
        "@nestjs/passport": "^7.1.0",
        "@nestjs/platform-express": "^7.4.4",
        "@nestjs/swagger": "^4.6.0",
        "@nestjs/typeorm": "^7.1.3",
        "aws-sdk": "^2.751.0",
        "bcrypt": "^5.0.0",
        "class-transformer": "^0.3.1",
        "class-validator": "^0.12.2",
        "mongodb": "^3.6.2",
        "nodemailer": "^6.4.11",
        "passport": "^0.4.1",
        "passport-jwt": "^4.0.0",
        "passport-local": "^1.0.0",
        "ramda": "^0.27.1",
        "reflect-metadata": "^0.1.13",
        "rxjs": "^6.6.3",
        "swagger-ui-express": "^4.1.4",
        "tsconfig-paths": "^3.9.0",
        "typeorm": "^0.2.26"
},
```
 
For Tooling issues:
- Node version: 12.18.1
- Platform: Windows

**Comment by jmcdo29** [comment]
Please provide a minimum reproduction repository.

**Comment by slowestmonkey** [comment]
The issue has been resolved. I installed that dependency via `npm i nestjs/mapped-types `(without @):
It left following entry in package.json:
```
@nestjs/mapped-types": "github:nestjs/mapped-types
```
and following entry in package-lock.json: 
```
@nestjs/mapped-types": {
      "version": "0.1.0",
      "resolved": "github:nestjs/mapped-types#a696c1f58ff8464b7c9c63c5a173d231554cc8df"
},
```
Changing dependency to version 0.1.0 and deletion of package-lock.json file and following reinstalling of dependencies fixes the problem.
